packages/flutter/test/material/chip_test.dart

```diff
@@ -216,43 +223,6 @@ Widget _chipWithOptionalDeleteButton({
   );
 }
 
-bool offsetsAreClose(Offset a, Offset b) => (a - b).distance < 1.0;
-bool radiiAreClose(double a, double b) => (a - b).abs() < 1.0;
-
-// Ripple pattern matches if there exists at least one ripple
-// with the [expectedCenter] and [expectedRadius].
-// This ensures the existence of a ripple.
-PaintPattern ripplePattern(Offset expectedCenter, double expectedRadius) {
-  return paints
-    ..something((Symbol method, List<dynamic> arguments) {
-        if (method != #drawCircle)
-          return false;
-        final Offset center = arguments[0] as Offset;
-        final double radius = arguments[1] as double;
-        return offsetsAreClose(center, expectedCenter) && radiiAreClose(radius, expectedRadius);
-      }
-    );
-}
-
-// Unique ripple pattern matches if there does not exist ripples
-// other than ones with the [expectedCenter] and [expectedRadius].
-// This ensures the nonexistence of two different ripples.
-PaintPattern uniqueRipplePattern(Offset expectedCenter, double expectedRadius) {
-  return paints
-    ..everything((Symbol method, List<dynamic> arguments) {
-        if (method != #drawCircle)
-          return true;
-        final Offset center = arguments[0] as Offset;
-        final double radius = arguments[1] as double;
-        if (offsetsAreClose(center, expectedCenter) && radiiAreClose(radius, expectedRadius))
-          return true;
-        throw '''
-              Expected: center == $expectedCenter, radius == $expectedRadius
-              Found: center == $center radius == $radius''';
-      }
-    );
-}
-
```

Comment:
A maximally robust version of this, with better debugging, is now in `ink_paint_test_utils`, and should be referenced for any ripple pattern paint testing